Kiosk/Bid Fulfillment/BidCheckingNetworkModel.swift

```diff
@@ -33,48 +54,35 @@ class BidCheckingNetworkModel: NSObject {
         // If polling fails, we can still show bid confirmation. Do not error.
         } .catchTo( RACSignal.empty() )
     }
-
-    private func pollForUpdatedSaleArtwork() -> RACSignal {
-
-        let beginningBidCents = fulfillmentController.bidDetails.saleArtwork?.saleHighestBid?.amountCents ?? 0
-
-        let updatedSaleArtworkSignal = getUpdatedSaleArtwork().flattenMap { [weak self] (saleObject) -> RACStream! in
+    
+    private func pollForUpdatedBidderPosition(bidderPositionId: String) -> RACSignal {
+        let updatedBidderPosition = getUpdatedBidderPosition(bidderPositionId).flattenMap { [weak self] bidderPositionObject in
             self?.pollRequests++
-
+            
             logger.log("Polling \(self?.pollRequests) of \(self?.maxPollRequests) for updated sale artwork")
-
-            let saleArtwork = saleObject as? SaleArtwork
-            let updatedBidCents = saleArtwork?.saleHighestBid?.amountCents ?? 0
-
+            
             // TODO: handle the case where the user was already the highest bidder
```

Comment:
Not sure, I guess that depends on what we would have wanted to happen. It seems to me that would just be the user increasing their max bid, but I don't think there's an issue with showing them the same message they would have seen anyway.